src/Drupal/DrupalDriverManager.php

```diff
@@ -64,7 +64,12 @@ public function registerDriver($name, DriverInterface $driver)
      */
     public function getDriver($name = null)
     {
-        $name = strtolower($name) ?: $this->defaultDriverName;
+        if (null === $name) {
+            $name = $this->defaultDriverName;
+        }
+        else {
+            $name = strtolower($name);
+        }
```

Comment:
We could keep the ternary and comply with the coding standards.
```suggestion
        $name = null === $name ? $this->defaultDriverName : strtolower($name);
```